assets/translations/en.json

```
@@ -313,5 +313,13 @@
   "resetPassword": "Reset password",
   "passwordMatchError": "Passwords do not match",
   "otpEmptyError": "Please enter the OTP code",
-  "otpInvalidError": "OTP must contain only numbers"
+  "otpInvalidError": "OTP must contain only numbers",
+  "startSignUp": "Start Sign Up",
+  "verificationCode": "Verification Code",
+  "enterCode": "Enter code",
+  "submitCode": "Submit Code",
+  "resendCode": "Resend Code",
```

```suggestion
  "startSignUp": "Start sign up",
  "verificationCode": "Verification code",
  "enterCode": "Enter code",
  "submitCode": "Submit code",
  "resendCode": "Resend code",
```

always using consistent capitalization. (Sentence case)